Student.java

```diff
@@ -1,10 +1,12 @@
 import greenfoot.*;  // (World, Actor, GreenfootImage, Greenfoot and MouseInfo)
 import java.util.ArrayList;
 /**
- * Write a description of class Student here.
+ * This class contains the animation for the student when the user runs the
+ * program
  * 
- * @author (your name) 
- * @version (a version number or a date)
+ * Kaushik Pendiyala 
+ * Version 1.0
+ * August 7, 2020
```

Comment:
Can you undo these changes? We are trying to avoid changes to `Student.java` to prevent conflicts with other people. Other than that, it looks good!